tests/.infra/crw-ci/dashboard-next/pr-check/Jenkinsfile

```diff
@@ -197,10 +179,8 @@ pipeline {
                 withCredentials([string(credentialsId: 'ed71c034-60bc-4fb1-bfdf-9570209076b5', variable: 'maxura_docker_password')]) {
                     sh """
                         echo "Build 'che-dashboard' image"
-                        echo "${maxura_docker_password}" | docker login -u maxura --password-stdin
```

Comment:
@sleshchenko: we need to log into docker, to avoid docker pull rate limits:
https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/blue/organizations/jenkins/dashboard-next-happy-path/detail/dashboard-next-happy-path/538/pipeline/ 
```
+ docker build . -f apache.Dockerfile -t quay.io/crw_pr/che-dashboard:204

Sending build context to Docker daemon  16.47MB


Step 1/14 : FROM docker.io/node:12.20.1-alpine3.12 as builder

toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```

Reply:
Ah, I see. I missed that docker io is used as based image. I'll fix it

Reply:
Returned back